server/models/backup.js

```diff
@@ -1,46 +1,53 @@
 'use strict';
 const Joi = require('joi');
-const MongoModels = require('hicsail-mongo-models');
+const Assert = require('assert');
+const AnchorModel = require('../anchor/anchor/anchor-model');
```

Comment:
to many `anchor/`

Reply:
made a mistake when branching off, need to delete this branch